graphjet-core/src/main/java/com/twitter/graphjet/hashing/SmallArrayBasedLongToDoubleMap.java

```diff
@@ -171,6 +171,24 @@ public boolean trim(int inputCapacity) {
   }
 
   /**
+   * @param key the input capacity that clients want to trim the map to
+   * @return true if the key is in the Map.
+   */
+  public boolean contains(long key) {
```

Comment:
Thanks for catching that. Also, the function is "constant" time, since it either scans linearly at most 8 entries or it does a set lookup if the size is more than 8.